src/EthereumHelpers.js

```diff
@@ -288,6 +300,15 @@ async function callWithRetry(
   })
 }
 
+/**
+ * @typedef {Object} DeployedContract
+ * @property {number|undefined} deployedAtBlock Number of block when contract was
+ * deployed.
+ *
+ * @typedef {Web3Contract & DeployedContract} Contract web3.eth.Contract enhanced
+ * with property containing block number when contract was deployed.
+ */
```

Comment:
We should probably define this up top. I would also suggest we name this combined type `DeployedContract` so our imports elsewhere are clearer, and maybe name our adjusted object `ContractDeploymentMixin` or similar.